Catappult/appcoins-wallet-android
# [feature/APPC-2472] Change wallet fiat

**What does this PR do?**

   Adds the ability to change the fiat currency displayed on the wallet.
   Can be changed from the home (the arrow close to the balance) or in the settings

**Database changed?**

   Yes 

**Where should the reviewer start?**

- ChangeFiatCurrencyFragment.kt
- ChooseCurrencyBottomSheetFragment.kt
- SettingsFragment.kt
- HomeFragment.kt
- FiatCurrenciesRepository.kt
- SettingsCurrencyPreference.kt

**How should this be manually tested?**

  Open the wallet, try changing the currency from within the home fragment, go to settings, check currency displayed and flag, change the currency from the settings.

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2472


**Code Review Checklist**
